stabilityai/pytorch/inference/docker/1.13/py3/cu117/Dockerfile.gpu

```diff
@@ -0,0 +1,10 @@
+FROM 763104351884.dkr.ecr.us-west-2.amazonaws.com/pytorch-inference:1.13.1-gpu-py39-cu117-ubuntu20.04-sagemaker as sagemaker
```

Comment:
For simplicity, let's just choose one framework version to release initially. This will make it easier to develop and identify issues with tests

Reply:
> For simplicity, let's just choose one framework version to release initially. This will make it easier to develop and identify issues with tests

Yeah, I am going to remove 1.13 after some final testing to confirm nothing needs it.